python-aws-lambda/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM lambci/lambda:build-python2.7 AS img1
+ 
+WORKDIR /app
+
+ADD . . 
+
+# Install app dependencies
+RUN pip install --upgrade pip
+RUN pip install rook -t .
+
+FROM kramos/alpine-zip AS img2
+
+RUN apk add --update zip
```

Comment:
try to install/use zip in lambda:build-python img, ok if doesnt work

Reply:
DIdn't work
But I changed the path of some lines to root insted of `/../../`